libs/wingc/src/lsp/completions.rs

```diff
@@ -279,7 +281,7 @@ fn get_completions_from_class(
 		.get_env()
 		.iter(true)
 		.filter_map(|symbol_data| {
-			if symbol_data.0 == "init" {
+			if symbol_data.0 == CLASS_INIT_NAME || symbol_data.0 == CLASS_INFLIGHT_INIT_NAME {
```

Comment:
curious: why the special case for `init` methods?

Reply:
You can't really call the constructor (`super` will be a different case) so there's no reason for it to be in the completion list.